app/controllers/api/braintree_controller.rb

```diff
@@ -37,11 +38,15 @@ def subscription_options
     {
       amount: params[:amount].to_f,
       plan_id: '35wm',
-      payment_method_token: Payment.customer(params[:email]).default_payment_method.token
+      payment_method_token: default_payment_method_token
     }
   end
 
   def braintree
     PaymentProcessor::Clients::Braintree
   end
+
+  def default_payment_method_token
+    @token ||= ::Payment.customer(params[:email]).try(:card_vault_token)
```

Comment:
> I thought it'd make sense to make whichever payment method is used by the user the default method, but I understand if this isn't the way you guys want to do it.

Presently, we only store (locally, and on the vault) a single payment method for a customer. Locally, we're storing the payment method's token on `Payment::BraintreeCustomer`. 

Above, I've kept your method name - `default_payment_method_token` , in case we chose to store multiple payment methods for a user (can only happen when users are able to authenticate) in the future.